src/esm_parser/esm_parser.py

```diff
@@ -1467,9 +1497,7 @@ def resolve_basic_choose(config, config_to_replace_in, choose_key, blackdict={})
 
     # Resolve the choose variables
     if choice in choices_available:
-        for update_key, update_value in six.iteritems(
-            choices_available[choice]
-        ):
+        for update_key, update_value in six.iteritems(choices_available[choice]):
```

Comment:
We should start cleaning up and remove the `six` stuff. Nothing uses Python 2 anymore.

```suggestion
        for update_key, update_value in choices_available[choice].items():
```